main.go

```diff
@@ -95,6 +97,18 @@ func main() {
 			fmt.Println("Oops, an error occurred! Rolling back...")
 			smug.Stop(*config, options, context)
 		}
+	case CommandCreate:
+		fmt.Printf("Creating %s...\n", options.Project)
```

Comment:
Also, let's remove those prints here. These are not long-running commands, so we don't need to print anything